modules/admin_manual/pages/release_notes.adoc

```diff
@@ -30,6 +31,130 @@
+
+=== New Enterprise Trial Mechanism
+
+To facilitate the evaluation of ownCloud Enterprise functionality, Server 10.5 comes with a new trial mechanism. Previously, the process to upgrade from a Community installation to an Enterprise trial was not that easy.
+
+With Server 10.5 this process has been changed and made a lot easier: Along with the changes to the Marketplace and in delivery, Enterprise functionality is basically available in every installation after upgrading or installing, respectively. The apps can be enabled like other apps which will start a 24-hour grace period if no valid license key is present. During this time the functionality can fully be used, even without a valid license key. To extend the period to 30 days, a demo license key can be obtained from a https://owncloud.com/try-enterprise/[new landing page]. The administrator will be informed and guided to obtain the demo license and can enter it in the web interface. If desired, the location of the landing page can be customized using the _config.php_ option `'grace_period.demo_key.link' => 'https://owncloud.com/try-enterprise/'`.
+
+=== New Admin UI to Supply License Keys
+
+As mentioned above, Server 10.5 adds new UI elements to set license keys in the _Admin -> General_ settings. Keys added this way will be stored in the database, not in _config.php_ as before. Still, there is legacy support taking into account when keys have been stored in _config.php_ or _license.config.php_.
+
+=== New Background Job for Change Detection in Federated Shares
+
+With ownCloud Server 10.2.1 a https://doc.owncloud.com/server/admin_manual/release_notes.html#background-job-for-change-detection-of-nested-federated-shares[background job for change detection of nested federated shares] has been added (`occ incoming-shares:poll`) to allow ownCloud Server to discover changes in federated shares in order to make them available for synchronization with the ownCloud Clients. Based on feedback a new, improved background job with more configuration options was added to Server 10.5. It replaces the former occ command which is now **deprecated** and should not be used anymore after upgrading to 10.5.
+
+In addition to discovering changes ("check"), the new background job also synchronizes meta data changes between involved servers ("scan") making them available without requiring users to actively browse them. 
+
+The new background job provides some configuration options to optimize its performance, especially for larger environments:
+- Minimum amount of hours since the last login of a user for that a scan is triggered (limits the execution of discovery & meta data sync to active users which have logged in during the configured time frame) (default: 24h)
```

Comment:
```suggestion
- Minimum amount of hours since the last login of a user that a scan is triggered (limits the execution of discovery & meta data sync to active users which have logged in during the configured time frame) (default: 24h)
```